README.md

```diff
@@ -0,0 +1,16 @@
+# Menge
+
+
+## Mac OSX
+
+Steps to follow in order to get `Menge` compiled:
```

Comment:
I use `homebrew` too. What packages, apart from `cmake`, did you need to install just for this?